src/importexport/musicxml/tests/tst_mxml_io.cpp

```diff
@@ -262,6 +264,17 @@ static void fixupScore(Score* score)
     score->setSaved(false);
 }
 
+static void checkScore(Score* score)
+{
```

Comment:
This is needed as there's no way to verify purely by means of .xml reference files whether the score is being imported expected.
There's probably an argument for using .mscx reference files instead, but I didn't want to make more work than was necessary at this point.